src/Components/Artwork/Details/BidTimerLine.tsx

```diff
@@ -17,7 +17,7 @@ export const BidTimerLine: React.FC<BidTimerLineProps> = ({ artwork }) => {
   const { lotClosesAt, registrationEndsAt, onlineBiddingExtended } =
     collectorSignals?.auction ?? {}
   const { time } = useTimer(lotClosesAt ?? "")
-  const { days, hours, minutes } = time
+  const { days, hours, minutes, seconds } = time
   const { isAuctionArtwork } = useArtworkGridContext()
   const biddingEnded = lotClosesAt && new Date(lotClosesAt) <= new Date()
```

Comment:
Unrelated to showing seconds, but since it's here - this bidding closed check needs an update as well away from `new Date()`. Up to you if you want to put that in its own PR!

Reply:
Yep. Would like to address all these client side time comparisons separately!